lte/gateway/c/core/oai/tasks/mme_app/mme_app_main.c

```diff
@@ -520,6 +522,7 @@ static void* mme_app_thread(__attribute__((unused)) void* args) {
 
   // Service started, but not healthy yet
   send_app_health_to_service303(&mme_app_task_zmq_ctx, TASK_MME_APP, false);
+  start_stats_timer();
```

Comment:
I had to move the start to this worker thread instead of the main thread as otherwise there is a race with zmq starting.

Reply:
It belongs to mme_app thread anyhow, so this is indeed the right location.